app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  def self.recent_three
+    order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] Great job adding the method for recent three posts but I highly recommend adding the needed associations here between the three tables so user could has many posts and many likes and many comments so you should add these relations with adding the foreign key on the many posts relation 🙏🏻🔥 so kindly add them here is User model 🙏🏻👍🏻✨

- [ OPTIONAL ]   Although this is optional I'd highly recommend you rename the method to something like recent_posts r as the current name is a bit misleading because you want to display the most three recent posts. ♻💪🏻